openmetadata-docs/content/connectors/custom-connectors.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Custom Connectors
+slug: /connectors/custom-connectors
+---
+
+# Custom Connectors
+
+Each of the services support providing a Custom Connector. It should be a Python class available from the Ingestion
+image. It should also match specific constraints on the methods to implement and how to send the Entities to be
```

Comment:
should we generalize that to something like "where your ingestion workflows are running (e.g. Docker Image, EC2 instance, etc.)"